### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thatguymehul**

### CODE REVIEW

**Comment by rettinghaus** [comment]
Looks like a Hacktoberfest contribution. Why not continue with a capital letter after the full stop?

**Comment by thatguymehul** [comment]
Please aproove, I would be forever grateful

File: README.md [hunk 1]
```diff
@@ -56,12 +56,12 @@ The following libraries are embedded in Verovio:
 Contributing
 ------------
 
-If you are interested in contributing to Verovio, please read the [contribution](doc/contributing.md) and [coding style](doc/guidelines.md) guidelines.
+If you are interested in contributing to Verovio we will be glad to accept your improovement all the time. please read the [contribution](doc/contributing.md) and [coding style](doc/guidelines.md) guidelines.
```
Comment (ahankinson) [inline README.md:59]: Improvement is spelled wrong.

File: README.md [hunk 2]
```diff
@@ -56,12 +56,12 @@ The following libraries are embedded in Verovio:
...
 
 Example output
 --------------
 
-The sample page of music shown below was generated with version 2.4.0-dev-2748fed
+The sample page of the music shown below has been generated with version 2.4.0-dev-2748fed
```
Comment (ahankinson) [inline README.md:64]: These changes are not grammatically correct.

File: README.md [hunk 3]
```diff
@@ -79,10 +79,10 @@ Example resources using verovio
 | [Music Sheet Viewer](http://www.partitionnumerique.com/music-sheet-viewer-wordpress-plugin) | tool | WordPress plugin for displaying graphical music from MEI data | 
 
 
-Digital score repositories on Github
+The Best Digital score repositories on Github
```
Comment (ahankinson) [inline README.md:82]: "Best" is subjective, and this list only provides a few examples and is not comprehensive.

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
 [![Build status](https://ci.appveyor.com/api/projects/status/121cxhmtwurxffh0?svg=true)](https://ci.appveyor.com/project/LaurentPugin/verovio-r1t6l)
 
-Verovio is a fast, portable and lightweight library for engraving [Music Encoding Initiative (MEI)](http://www.music-encoding.org) digital scores into SVG images. Verovio also contains on-the-fly converters to render [Plaine & Easie Code](https://www.iaml.info/plaine-easie-code), [Humdrum](https://www.humdrum.org), [Musedata](https://musedata.org), [MusicXML](https://www.musicxml.com), [EsAC](http://esac-data.org), and [ABC](https://en.wikipedia.org/wiki/ABC_notation) digital scores. 
+Verovio is a fast, portable and lightweight library for engraving and digitalizing [Music Encoding Initiative (MEI)](http://www.music-encoding.org) digital scores into SVG images. Verovio also contains on-the-fly converters to render [Plaine & Easie Code](https://www.iaml.info/plaine-easie-code), [Humdrum](https://www.humdrum.org), [Musedata](https://musedata.org), [MusicXML](https://www.musicxml.com), [EsAC](http://esac-data.org), and [ABC](https://en.wikipedia.org/wiki/ABC_notation) digital scores.
```
Comment (ahankinson) [inline README.md:5]: Verovio uses pre-digitized scores, so it is not a tool for digitizing.

**Review by ahankinson — COMMENTED** [review]
👎 reject

**Comment by craigsapp** [comment]
All changes would have been grammerly incoreckt.

**Comment by ahankinson** [comment]
You spelled "wood" wrong!

**Comment by craigsapp** [comment]
> You spelled "wood" wrong!

Upps, me too speels not being goodly.

> Looks like a Hacktoberfest contribution. Why not continue with a capital letter after the full stop?

Hmmm, that is a real thing?
   https://blog.domenic.me/hacktoberfest

Do you get your free T-shirt for posting the PR or for it being accepted, @thatguymehul?

**Comment by samuelbradshaw** [comment]
No need to ridicule a contributor.

**Comment by craigsapp** [comment]
That was not a serious contributor, and his account has been deleted from Github.

**Pull request closed without merging**